### PR TITLE
CODENVY-470: Add user profile updated event

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Signed-off-by: Aleksandr Andrienko aandrienko@codenvy.com

### CODE REVIEW

**Comment by skabashnyuk** [comment]
Are you sure you need whole profile in event?

File: wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/notification/ProfileUpdatedEvent.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      * Returns copy updated profile
+     */
+    public Profile getProfile() {
+        return profile;
+    }
+
+    @Override
+    public String toString() {
+        return "ProfileUpdatedEvent{" +
+               "profile=" + profile +
```
Comment (benoitf) [inline wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/notification/ProfileUpdatedEvent.java:42]: Profile class has no toString() method so I'm not sure it is what is expected there

File: wsmaster/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserProfileServiceTest.java [hunk 2]
```diff
@@ -315,6 +320,7 @@ public void shouldBeAbleToUpdateCurrentProfileAttributes() throws Exception {
 
         assertEquals(response.getStatus(), OK.getStatusCode());
         verify(profileDao, times(1)).update(profile);
+        verify(eventService).publish(any());
```
Comment (benoitf) [inline wsmaster/che-core-api-user/src/test/java/org/eclipse/che/api/user/server/UserProfileServiceTest.java:323]: we should verify the profile is inside the event no ? call to getProfile() on the event

**Comment by skabashnyuk** [comment]
Consider to remove Profile from event. 
It can be replaced with userid for instance.
No GO in current state.

**Comment by AndrienkoAleksandr** [comment]
Code will be reworked.

**Pull request closed without merging**